src/views/consumer/list.jsx

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import Layout from '../components/layouts/Publisher';
 import Pagination from '../components/Pagination';
 
+// TODO: This page is not linked to from anywhere
+// so we should probably remove it.
```

Comment:
It was from before the topics became the main way to find datasets, but it's quite handy to be able to just list published datasets even if we don't link to it, so I would keep it.